modules/cluster-logging-collector-log-forwarding-about.adoc

```diff
@@ -123,11 +126,6 @@ spec:
 ** The `outputRefs` is the name of the output to use.
 ** Optional: String. One or more labels to add to the logs.
 
-[discrete]
-[id="cluster-logging-external-fluentd_{context}"]
-== Fluentd log handling when the external log aggregator is unavailable
```

Comment:
Do we have a section elsewhere talking about this topic related to the collector?  Parts of this are still accurate though reading this blob the PVC bit is completely incorrect

Reply:
Can you please suggest an edit? Does Fluentd require to be changed to something else?